pkg/operator/resource/resourceapply/generic.go

```diff
@@ -25,6 +24,7 @@ var (
 
 func init() {
 	utilruntime.Must(api.InstallKube(genericScheme))
```

Comment:
Is GH hiding from me where `genericScheme` was defined before?

Reply:
> Is GH hiding from me where `genericScheme` was defined before?

same file, higher up.

Reply:
genericScheme = runtime.NewScheme() above